File: c++/1.cpp
#include<iostream>
using namespace std;
int main(){
    int rows,cols;
    cout<<"Enter the no. of rows : ";
    cin>>rows;
    cout<<"Enter the no. of cols : ";
    cin>>cols;

    //Normal Rectangle
    for(int i=1;i<=rows;i++){
        for(int j=1;j<=cols;j++){
            cout<<"* ";
        }
        cout<<endl;
    }

    //For Hollow Rectangle
    cout<<"==============================================\n";
    for(int i=1;i<=rows;i++){
        for(int j=1;j<=cols;j++){
            if(i==1||i==rows||j==1||j==cols)
            cout<<"* ";
            else
            cout<<"  ";
        }
        cout<<endl;
    }   
}

File: python/1.py

rows=int(input("Enter rows : "))
cols=int(input("Enter cols : "))

#Rectangle
for i in range(rows):
    for j in range(cols):
        print("* ",end="")
    print()    

#Hollow Rectangle
print("-------------------------------") 

for i in range(1,rows+1):
    for j in range(1,cols+1):
        if(i==1 or i==rows or j==1 or j==cols):
            print("* ",end="")
        else:
            print("  ",end="")  
    print()          